src/viser/_messages.py

```diff
@@ -65,6 +65,15 @@ class CameraFrustumMessage(Message):
     image_base64_data: Optional[str]
 
 
+@dataclasses.dataclass
+class GlTFMessage(Message):
+    """GlTF Message"""
+
+    name: str
+    gltf_base64_data: str
```

Comment:
Is it possible to use `bytes` here directly? We use base64 for images because they make data URIs easier, but we should have no trouble serializing raw bytes.